src/lib/openapi/spec/context-field-schema.ts

```diff
@@ -5,41 +5,59 @@ export const contextFieldSchema = {
     $id: '#/components/schemas/contextFieldSchema',
     type: 'object',
     additionalProperties: false,
+    description:
+        'A [context field](https://docs.getunleash.io/reference/unleash-context) used to configure strategies with data used to evaluate isEnabled calls',
```

Comment:
This sentence runs on a bit and I'm not sure what it's saying. Would it be fine to shorten it down a bit like this?

```suggestion
        'A representation of a [context field](https://docs.getunleash.io/reference/unleash-context).',
```

Reply:
Yes, thank you. The link should cover what a context field actually is.